packages/amazonq/README.md

```diff
@@ -1,22 +1,37 @@
-## Inline code suggestions
+[![Twitter Follow](https://img.shields.io/twitter/follow/aws?logo=twitter&style=social)](https://twitter.com/awscloud)
+[![Youtube Channel Views](https://img.shields.io/youtube/channel/views/UCd6MoB9NC6uYN2grvUNT-Zg?style=social)](https://www.youtube.com/@amazonwebservices)
+[![Try Amazon Q free trial](https://img.shields.io/badge/Try%20Amazon%20Q-Free%20trial-success)](https://aws.amazon.com/q/developer/?editor=vscode)
+![Marketplace Downloads](https://img.shields.io/vscode-marketplace/d/AmazonWebServices.amazon-q-vscode.svg)
+
+# Inline code suggestions
 
 Code faster with inline code suggestions as you type.
 
-![Inline code suggestion demo](https://github.com/aws/aws-toolkit-vscode/raw/HEAD/docs/marketplace/vscode/amazonq/inline.gif)
+# Getting Started
+
+**Free Tier** - create or log in with an AWS Builder ID (no AWS account needed!).
+
+**Pro Tier** - if your organization is on the Amazon Q Developer Pro tier, log in with single sign-on.
+
+![Authentication gif](https://raw.githubusercontent.com/aws/aws-toolkit-vscode/HEAD/docs/marketplace/vscode/amazonq/auth-Q.gif)
+
+# Features
 
-<sub>[_15+ languages supported including Python, TypeScript, Rust, Terraform, AWS Cloudformation, and more_](https://docs.aws.amazon.com/amazonq/latest/qdeveloper-ug/q-language-ide-support.html)</sub>
```

Comment:
`<sub>` is still in other parts of this doc, can we remove it there too. since we are changing stuff anyway.

Reply:
Done.